build.gradle

```diff
@@ -56,15 +64,23 @@ subprojects {
     dependencies {
         implementation "com.github.ben-manes.caffeine:caffeine:${depVersions.caffeine}"
         implementation "com.commercetools.sdk.jvm.core:commercetools-java-client:${depVersions.commercetoolsSdkJvm}"
+        constraints {
+            implementation "io.netty:netty-codec-http:${depVersions.netty}"
+            implementation "io.netty:netty-transport-native-epoll:${depVersions.netty}"
+        }
         implementation "com.commercetools.sdk.jvm.core:commercetools-models:${depVersions.commercetoolsSdkJvm}"
         implementation "com.commercetools.sdk.jvm.core:commercetools-convenience:${depVersions.commercetoolsSdkJvm}"
+        constraints {
+            implementation "commons-io:commons-io:${depVersions.commons}"
```

Comment:
Did you try to update the library that we use? I guess these deps are transitative ones and we don't use them directly?

Reply:
Updated the direct dependency versions even they are not reported
https://github.com/commercetools/commercetools-payone-integration/pull/371/commits/fe019b82356713c6b2b56fd18d45e28e152d78d0

> I guess these deps are transitative ones and we don't use them directly
So your idea is to exclude these transitive dependency rather than update them?

Reply:
My idea was if it's possible, upgrade the deps we're using in our library, not the ones that we're not directly using.

Reply:
If not, can you leave the issue for the SDKs so they upgrade the deps.

Reply:
@lojzatran Have checked in jvm-sdk already. Their latest version doesn't apply the latest version of commons-io, and so do other vulnerable dependencies.

We can create an issue in jvm-sdk side, but meanwhile we can fix it in our gradle build until they publish a newer release to resolve the vulnerability cases.

Reply:
Yep, that's why I already approved the PR :)

Reply:
Corresponding issue has been created in jvm-sdk
https://github.com/commercetools/commercetools-jvm-sdk/issues/2137